frontend/src/features/map/layers/InterestPointLayer.tsx

```diff
@@ -338,4 +328,42 @@ export function InterestPointLayer({ map }: BaseMapChildrenProps) {
       ) : null}
     </div>
   )
+
+  function shouldStyledLines(feature) {
```

Comment:
comme la fonction renvoie un booléen tu pourrais la préfixer par `is`, genre `isLineFeature` ou `getisLineFeature`, t'en penses quoi? on essaye d'avoir une convention pour les booléen (variables et fonctions).
Je comprends bien ta façon de nommer mais sur env on part plutôt sur l'idée que le nom d'une fonction reflète ce qu'elle fait/ doit renvoyer et non pourquoi elle est appelée (ici t'es plutôt parti la dessus on dirait)

Reply:
Je nomme les fonctions qui renvoie des booléens avec des "yes/no questions" (si tu te rappelles de tes cours d'anglais du collège). Donc j'ai tendance à nommer ces méthodes avec des is/has/should. Après si c'est une convention c'est pas un pb

Reply:
Ce que je veux dire c'est que la fonction te permet de savoir si la feature est une ligne ou non, pas de savoir si on doit la stylisée. Si cette fonction était utilisée ailleurs que pour le style (par exemple savoir si on affiche ou non un truc si la feature est une ligne) le nom de la fonction ne serait plus très pertinent, tu vois ce que je veux dire?

Reply:
yes je vois, je modifie 👍